### ISSUE TITLE
Report expenses, group, stats, and submit reimbursement request for Techogee event | 2018-05-21

### ISSUE THREAD

**Issue by jsonthebson** [techlahoma/user-groups#562]
What: Techogee - Muskogee's Monthly Tech User Meetup 
When: 2018-05-21 6:00 PM
Where: 801 W Okmulgee St, Muskogee

Check meetup for RSVP count: https://www.meetup.com/Techogee/events/246462432/

To request reimbursement, please complete the following steps:
- [ ] Picture of receipt(s)
- [ ] Final head count
- [ ] Photo of meetup in action
- [ ] Submit [request for reimbursement](https://github.com/techlahoma/user-groups/wiki/Requesting-Reimbursement) from $techlahoma on https://cash.me 

cc @sgelbart, @ezrinjaz, @lemoney

**Comment by kacollins** [comment]
Was this event cancelled? It's not on Meetup anymore.

**Comment by ezrinjaz** [comment]
I don't think we set it up on Meetup.com. I think we only scheduled through April there, and I forgot we had moved it to the 21st here because of Memorial Day so we didn't meet.
